error 190919-ensure-axiom-immutable.pyv:8:95: Only immutable relations can be referenced in this context
program has resolution errors.
mypyvy exiting with status 1
